document/core/exec/runtime.rst

```diff
@@ -7,14 +7,14 @@ Runtime Structure
 :ref:`Store <store>`, :ref:`stack <stack>`, and other *runtime structure* forming the WebAssembly abstract machine, such as :ref:`values <syntax-val>` or :ref:`module instances <syntax-moduleinst>`, are made precise in terms of additional auxiliary syntax.
 
 
-.. index:: ! value, constant, value type, integer, floating-point
+.. index:: ! value, constant, value type, integer, floating-point, fixed-width simd
    pair: abstract syntax; value
 .. _syntax-val:
 
 Values
 ~~~~~~
 
-WebAssembly computations manipulate *values* of the four basic :ref:`value types <syntax-valtype>`: :ref:`integers <syntax-int>` and :ref:`floating-point data <syntax-float>` of 32 or 64 bit width each, respectively.
+WebAssembly computations manipulate *values* of the five basic :ref:`value types <syntax-valtype>`: :ref:`integers <syntax-int>` and :ref:`floating-point data <syntax-float>` of 32 or 64 bit width each  respectively, and :ref: `SIMD data <syntax-simd>` of 128 bit width representing packed integers of floating-point values.
```

Comment:
We should define "packed integers" explicitly. I thought it may have been used in the memory instruction section, but that just refers to "storage size" which is not quite what we want here.

Reply:
I agree, just not sure where the right place for them would be. I'm wondering if just having them be in document/core/syntax/values.rst under a `Representations` section make sense? I initially thought not, because it's mostly addressing the structure of types - but it would make sense to have them there because <syntax-simd> would point to there.

Reply:
@rossberg thoughts?

Reply:
@rossberg Explicitly linking from #195 to address syntax issue - looking for feedback on how to address the different representations better here so that can be incorporated into subsequent PRs. I'm looking for the right place to express different representations with the general syntax of `{t}{lane_width}x{lanes}` where `t` is the type, `lane_width` is the number of bits per packed value, and `lanes` is the number of packed values.

Reply:
The definition of iN in syntax/values is generic in N, so already defines i8 and i16. Likewise, all the ops in the numerics section are generic.

See comment below regarding abstract syntax of SIMD literals. The definition here has to match the abstract syntax of the const instruction.